pkg/build/api/v1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,1079 @@
+				}
+			}
+		} else {
+			out.Items = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_v1_BuildLog(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*BuildLog)
+		out := out.(*BuildLog)
+		out.TypeMeta = in.TypeMeta
+		return nil
+	}
+}
+
+func DeepCopy_v1_BuildLogOptions(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*BuildLogOptions)
+		out := out.(*BuildLogOptions)
+		out.TypeMeta = in.TypeMeta
+		out.Container = in.Container
+		out.Follow = in.Follow
+		out.Previous = in.Previous
+		if in.SinceSeconds != nil {
+			in, out := &in.SinceSeconds, &out.SinceSeconds
+			*out = new(int64)
+			**out = **in
+		} else {
+			out.SinceSeconds = nil
+		}
+		if in.SinceTime != nil {
+			in, out := &in.SinceTime, &out.SinceTime
+			*out = new(unversioned.Time)
+			**out = (*in).DeepCopy()
+		} else {
+			out.SinceTime = nil
+		}
+		out.Timestamps = in.Timestamps
+		if in.TailLines != nil {
+			in, out := &in.TailLines, &out.TailLines
+			*out = new(int64)
+			**out = **in
+		} else {
+			out.TailLines = nil
+		}
+		if in.LimitBytes != nil {
+			in, out := &in.LimitBytes, &out.LimitBytes
+			*out = new(int64)
+			**out = **in
+		} else {
+			out.LimitBytes = nil
+		}
+		out.NoWait = in.NoWait
+		if in.Version != nil {
+			in, out := &in.Version, &out.Version
+			*out = new(int64)
+			**out = **in
+		} else {
+			out.Version = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_v1_BuildOutput(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*BuildOutput)
+		out := out.(*BuildOutput)
+		if in.To != nil {
+			in, out := &in.To, &out.To
+			*out = new(api_v1.ObjectReference)
+			**out = **in
+		} else {
+			out.To = nil
+		}
+		if in.PushSecret != nil {
+			in, out := &in.PushSecret, &out.PushSecret
+			*out = new(api_v1.LocalObjectReference)
+			**out = **in
+		} else {
+			out.PushSecret = nil
+		}
+		return nil
+	}
+}
+
+func DeepCopy_v1_BuildPostCommitSpec(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*BuildPostCommitSpec)
+		out := out.(*BuildPostCommitSpec)
+		if in.Command != nil {
+			in, out := &in.Command, &out.Command
+			*out = make([]string, len(*in))
+			copy(*out, *in)
+		} else {
+			out.Command = nil
+		}
+		if in.Args != nil {
+			in, out := &in.Args, &out.Args
+			*out = make([]string, len(*in))
+			copy(*out, *in)
+		} else {
+			out.Args = nil
+		}
+		out.Script = in.Script
+		return nil
+	}
+}
+
+func DeepCopy_v1_BuildRequest(in interface{}, out interface{}, c *conversion.Cloner) error {
+	{
+		in := in.(*BuildRequest)
+		out := out.(*BuildRequest)
+		out.TypeMeta = in.TypeMeta
+		if newVal, err := c.DeepCopy(&in.ObjectMeta); err != nil {
+			return err
+		} else {
+			out.ObjectMeta = *newVal.(*api_v1.ObjectMeta)
+		}
+		if in.Revision != nil {
+			in, out := &in.Revision, &out.Revision
+			if newVal, err := c.DeepCopy(*in); err != nil {
+				return err
+			} else {
+				*out = newVal.(SourceRevision)
```

Comment:
Looks like you tracked it down yourself yesterday.

You had hit a broken code path in the generator (
https://github.com/kubernetes/kubernetes/blob/bd7d7cc02a370180c25b20096b450b79ba2713b8/cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go#L612-L612): it must look like this:

```go
sw.Do("*out = newVal.($.|raw$)\n", t)
```

Will fix that upstream.